### DISCUSSION TITLE
keycloak setup accross data centers.

### DISCUSSION THREAD

**Discussion by Rohit-Rathor3** [keycloak/keycloak#33265] [Q&A]
Hi Everyone , i have come to a problem where i have to setup keycloak accross data centers.
Actually , i have two data centers , where we have two load balancers which will routes the traffic across the cluster in datacenter .
The limitation is that one load balancer can routes the traffic in only cluster of one data center. There is no connectivity accross the datacenter through load balancers. So is there any way to handle this problem at keycloak level only. Your discussion and suggetions will be admireable.

**Comment by codespearhead** [comment]
Duplicate of [1]. Please close this discussion [2].

[1] https://github.com/keycloak/keycloak/discussions/33264
[2] https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion
